react/features/video-status-label/components/VideoStatusLabel.js

```diff
@@ -93,10 +130,16 @@ export class VideoStatusLabel extends Component {
                 ? t('videoStatus.hd') : t('videoStatus.sd');
         }
 
-        const filmstripClassName
+        // Determine which classes should be set on the component. These classes
+        // will used to help with animations and setting position.
+        const baseClasses = 'video-state-indicator moveToCorner';
+        const filmstrip
             = _filmstripVisible ? 'with-filmstrip' : 'without-filmstrip';
+        const oneOnOne = _remoteVideosVisible && _remoteVideosCount
+            ? 'with-remote-videos' : 'without-remote-videos';
```

Comment:
Do we actually need both remoteVideosVisible and remoteVideosCount? Can you please explain a little bit about that check?